diff --git a/node_modules/@langchain/anthropic/dist/utils/message_inputs.cjs b/node_modules/@langchain/anthropic/dist/utils/message_inputs.cjs
index 640fd2c..5418b41 100644
--- a/node_modules/@langchain/anthropic/dist/utils/message_inputs.cjs
+++ b/node_modules/@langchain/anthropic/dist/utils/message_inputs.cjs
@@ -70,6 +70,7 @@ function* _formatContentBlocks(content, toolCalls) {
 		"text_editor_code_execution_tool_result",
 		"tool_result",
 		"tool_use",
+		"web_fetch_tool_result",
 		"web_search_result",
 		"web_search_tool_result"
 	];
diff --git a/node_modules/@langchain/anthropic/dist/utils/message_inputs.js b/node_modules/@langchain/anthropic/dist/utils/message_inputs.js
index 3af742b..15f65bd 100644
--- a/node_modules/@langchain/anthropic/dist/utils/message_inputs.js
+++ b/node_modules/@langchain/anthropic/dist/utils/message_inputs.js
@@ -69,6 +69,7 @@ function* _formatContentBlocks(content, toolCalls) {
 		"text_editor_code_execution_tool_result",
 		"tool_result",
 		"tool_use",
+		"web_fetch_tool_result",
 		"web_search_result",
 		"web_search_tool_result"
 	];
diff --git a/node_modules/@langchain/anthropic/dist/utils/message_outputs.cjs b/node_modules/@langchain/anthropic/dist/utils/message_outputs.cjs
index d005135..92907eb 100644
--- a/node_modules/@langchain/anthropic/dist/utils/message_outputs.cjs
+++ b/node_modules/@langchain/anthropic/dist/utils/message_outputs.cjs
@@ -43,6 +43,7 @@ function _makeMessageChunkFromAnthropicEvent(data, fields) {
 		"tool_use",
 		"document",
 		"server_tool_use",
+		"web_fetch_tool_result",
 		"web_search_tool_result"
 	].includes(data.content_block.type)) {
 		const contentBlock = data.content_block;
diff --git a/node_modules/@langchain/anthropic/dist/utils/message_outputs.js b/node_modules/@langchain/anthropic/dist/utils/message_outputs.js
index 5222261..78a8fe1 100644
--- a/node_modules/@langchain/anthropic/dist/utils/message_outputs.js
+++ b/node_modules/@langchain/anthropic/dist/utils/message_outputs.js
@@ -42,6 +42,7 @@ function _makeMessageChunkFromAnthropicEvent(data, fields) {
 		"tool_use",
 		"document",
 		"server_tool_use",
+		"web_fetch_tool_result",
 		"web_search_tool_result"
 	].includes(data.content_block.type)) {
 		const contentBlock = data.content_block;
